critwall/src/test/resources/cadc-storage-adapter-fs.properties

```diff
@@ -0,0 +1,11 @@
+# The root of the file system (as seen by the container)
```

Comment:
I don't think this file is needed since FileSyncJobTest constructs the adapter via the utility ctor... if it was used it would belong in intTest/resources.

Reply:
file is gone. was a remnant from having the FileSyncJobTest in the 'test' folder instead of 'intTest'